erpnext/accounts/report/general_ledger/general_ledger.html

```diff
@@ -1,8 +1,6 @@
 <h2 class="text-center">{%= __("Statement of Account") %}</h2>
 <h4 class="text-center">
-	{% if (filters.party_name) { %}
```

Comment:
This has been done purposefully, in case there is naming series for customer, it should show customer_name value instead of ID.